spec/models/pacbio/library_factory_spec.rb

```diff
@@ -114,20 +113,28 @@
       end
 
       context 'when save errors' do
+        subject(:factory) { described_class.new(attributes) }
+
         before do
-          @factory = described_class.new(attributes)
-          allow(@factory).to receive(:valid?).and_return true
+          # This test seems to be designed to catch edge cases, in which problems that
```

Comment:
I'm not sure what this means. Not clear why we need to stub out the valid method. Is it just a spurious test that can be removed?